File: android/app/src/main/java/vn/team/freechat/data/MessageSent.java
package vn.team.freechat.data;

import com.tvd12.ezyfoxserver.client.constant.EzyConstant;

import vn.team.freechat.constant.MessageType;

/**
 * Created by tavandung12 on 10/7/18.
 */

public class MessageSent extends Message {

    public MessageSent(long channelId, String message) {
        super( channelId, message);
    }

    @Override
    public EzyConstant getType() {
        return MessageType.SENT;
    }
}

File: android/app/src/main/java/com/tvd12/freechat/data/MessageSent.kt
package com.tvd12.freechat.data;

import com.tvd12.ezyfoxserver.client.constant.EzyConstant
import com.tvd12.freechat.constant.MessageType

/**
 * Created by tavandung12 on 10/7/18.
 */

class MessageSent(
    message: String
) : Message(message) {

    override  fun getType() : EzyConstant {
        return MessageType.SENT
    }
}